runtime/jcl/common/mgmtmempool.c

```diff
@@ -358,7 +358,7 @@ Java_com_ibm_java_lang_management_internal_MemoryPoolMXBeanImpl_resetPeakUsageIm
 		/* NonHeap MemoryPool */
 		J9MemorySegmentList *segList = getMemorySegmentList(javaVM, id);
 		if (NULL != segList) {
-			processSegmentList(env, NULL, NULL, segList, 0, &mgmt->nonHeapMemoryPools[id-J9VM_MANAGEMENT_POOL_NONHEAP_SEG].peakSize, &mgmt->nonHeapMemoryPools[id-J9VM_MANAGEMENT_POOL_NONHEAP_SEG].peakUsed, 2);
+			processSegmentList(env, NULL, NULL, segList, 0, 0, &mgmt->nonHeapMemoryPools[id-J9VM_MANAGEMENT_POOL_NONHEAP_SEG].peakSize, &mgmt->nonHeapMemoryPools[id-J9VM_MANAGEMENT_POOL_NONHEAP_SEG].peakUsed, 2);
```

Comment:
Is 0 the right default for max size here?  Some of the non-heap pools default to `-1` while others to jit codecache sizes.

Reply:
Hi Dan, processSegmentList() is called at above for resetting peak values only(action = 2), so pass 0 here because we are not going to use the parameter (for maxSize) for action =2 case.